parsers/svg-to-jsx/README.md

```diff
@@ -0,0 +1,199 @@
+| `wrapper.tag`  | optional     | `string`          |  | An HTML parent tag will be used to wrap the SVG tag.              |
+| `wrapper.className`  | optional     | `string`    |      | A string or a pattern used to set a className attribute on the potential parent tag wrapping the SVG tag.                           |
+
+
+## Types
+
+ℹ️ **Please be aware that, depending on the order you use parsers, their input and output types have to match.**
+
+### Input
+
+Array of object with at least the key `value.url` or `value.content`.
+
+```ts
+Array<
+  Record<string, any> & {
+    type: string;
+    name: string;
+    value: ({ url?: string } | { content?: string }) & { [key: string]: any };
+  }
+>
+```
+### Output
+
+Input type extended with `value.content` and `value.fileName`
+
+```ts
+Array<
+  Record<string, any> & {
+    type: string; 
+    value: { content: string; fileName: string; [key: string]: any }
+  }
+>;
+```
+
+
+## Basic usage
+### Config
+
+```json
+{
+  "name": "svg-to-jsx"
+}
+```
+
+### Before/After
+#### Input
+
+```json5
+{
+  "type": "vector",
+  "name": "activity.svg",
+  "value": {
+    "url": "https://s3-us-west-2.amazonaws.com/figma-alpha-api/img/99b5/b311/257c650341b701d691be78f247b9cf5e"
+    //...
+  }
+  //...
+}
+```
+
+#### Output
+
+ℹ️  The `value.content` is the content of the file. It will be written as a file if the path of the rule in your config file is a directory.
+Otherwise, it's a simple json object written in the file defined as a path.
+
+```
+{
+  "type": "vector",
+  "name": "activity",
+  "value": {
+    "fileName": "Activity.jsx",
+    "content": "export default () => (
+      <svg
+        width="40"
+        height="40"
+        viewBox="0 0 40 40"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M36.6667 20H30L25 35L15 5L10 20H3.33334"
+          stroke="black"
+          stroke-width="3.33333"
+          stroke-linecap="round"
+          stroke-linejoin="round"
+        />
+      </svg>
+    );",
+  },
+  ...
+}
+```
+
+## Complex usage
+### Config
+
+```json
+{
+  "name": "svg-to-jsx",
+  "options": {
+    "prepend": "import React from 'react';",
+    "variableFormat": "camelCase",
+    "wrapper": {
+      "tag": "div",
+      "className": "icon-{{name}} icon"
+    }
+  }
+}
+```
+
+### Before/After
+#### Input
+```json5
+{
+  "type": "vector",
+  "name": "activity.svg",
+  "value": {
+    "url": "https://s3-us-west-2.amazonaws.com/figma-alpha-api/img/99b5/b311/257c650341b701d691be78f247b9cf5e"
+    //...
+  }
+  //...
+}
+```
+#### Output
+
+ℹ️  The `value.content` is the content of the file. It will be written as a file if the path of the rule in your config file is a directory.
+Otherwise, it's a simple json object written in the file defined as a path.
```

Comment:
ℹ️ The `value.content` is the content of the file.
If the `path` of your rule targets a directory, the content will be written as a file name with the following pattern: `{vectorName}.jsx`.
If the `path` of your rule targets a file, the content will be written as a JSON object inside it.